packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -93,17 +122,37 @@ class ExpansionPanel {
 ///
 ///  * [ExpansionPanel]
 ///  * <https://material.google.com/components/expansion-panels.html>
-class ExpansionPanelList extends StatelessWidget {
+class ExpansionPanelList extends StatefulWidget {
   /// Creates an expansion panel list widget. The [expansionCallback] is
   /// triggered when an expansion panel expand/collapse button is pushed.
+  ///
+  /// The [children] and [animationDuration] arguments must not be null.
   const ExpansionPanelList({
     Key key,
     this.children = const <ExpansionPanel>[],
     this.expansionCallback,
-    this.animationDuration = kThemeAnimationDuration
-  }) : assert(children != null),
-       assert(animationDuration != null),
-       super(key: key);
+    this.animationDuration = kThemeAnimationDuration,
+  }) :  assert(children != null),
+        assert(animationDuration != null),
+        _allowOnlyOnePanelOpen = false,
+        super(key: key);
+
+  /// Creates a radio expansion panel list widget.
+  ///
+  /// This widget allows for at most one panel in the list to be open.
+  /// The expansion panel callback is triggered when an expansion panel
+  /// expand/collapse button is pushed. The [children] and [animationDuration]
+  /// arguments must not be null. The [children] objects also must of type
+  /// [ExpansionPanelRadio].
+  const ExpansionPanelList.radio({
+    Key key,
+    this.children = const <ExpansionPanelRadio>[],
```

Comment:
Something I missed earlier - this isn't a type annotation for children, it is just the type for the default value.  You need to use an explicitly typed param with an initializer.
```
({
  List<ExpansionPanelRadio> children = ..,
}) : children = children,
```